spec/integration/create_subscription_spec.rb

```diff
@@ -59,18 +65,14 @@ def create_subscription
     end
 
     context "with a parameter to skip sending the confirmation email" do
-      def create_subscription
-        post "/subscriptions", params: { subscriber_list_id: subscriber_list.id, address: subscriber.address, frequency: frequency, skip_confirmation_email: true }
-      end
-
       it "does not send a confirmation email" do
         stub_notify
-        create_subscription
+        create_subscription(extra_params: { skip_confirmation_email: true })
         expect(a_request(:post, /notifications/)).to_not have_been_made
       end
 
       it "returns status code 200" do
-        create_subscription
+        create_subscription(extra_params: { skip_confirmation_email: true })
```

Comment:
Yep, I prefer this way I think. It's immediately more obvious what the difference is.